### ISSUE TITLE
[Documentation] Comparison with nuklear

### ISSUE THREAD

**Issue by VisenDev** [nicbarker/clay#95]
[Nuklear](https://github.com/Immediate-Mode-UI/Nuklear) is another immediate mode header-only ui layout library. It would be really useful to see some sort of comparison between these two projects, and what use cases clay fulfills that nuklear does not.

**Comment by pratss10** [comment]
I'm interested in taking on this issue and exploring the similarities and differences between Nuklear and Clay. My approach will include identifying unique use cases that Clay fulfills, as well as areas where Nuklear might excel. I’ll aim to provide a detailed and actionable analysis.

Could you assign this issue to me? I’d be happy to share my findings once I've conducted some research.

**Comment by nicbarker** [comment]
@pratss10 Absolutely! I'd be happy to hear what you come up with, having not really used Nuklear myself 🙂
